gpt_index/docstore.py

```diff
@@ -68,18 +44,8 @@ def load_from_dict(
             elif doc_type == Node.get_type():
```

Comment:
so right now the type is either "Document" or whatever the Node is right?

Reply:
Yep, but this reminded me that we have to load different type of Nodes differently.